src/validation.h

```diff
@@ -206,12 +206,26 @@ struct PackageMempoolAcceptResult
 };
 
 /**
- * (Try to) add a transaction to the memory pool.
- * @param[in]  bypass_limits   When true, don't enforce mempool fee limits.
- * @param[in]  test_accept     When true, run validation checks but don't submit to mempool.
+ * Try to add a transaction to the mempool with a specified acceptance time. This is an internal
```

Comment:
"with a specified acceptance time" is redundant with the parameter documentation below.

Reply:
Removed in 9a0b0fc